### ISSUE TITLE
xWebAppPool Get-TargetResource fails

### ISSUE THREAD

**Issue by AceHack** [PowerShell/xWebAdministration#92]
Below are my settings, and whenever I run Get-DscConfiguration I get the following error.

            xWebAppPool InstallWebAppPool {
                Ensure = 'Present'
                Name = 'MyAppPool"
                State = 'Started'
                autoStart = 'true'
                managedRuntimeVersion = 'v4.0'
                managedPipelineMode = 'Integrated'
            }

Get-DscConfiguration : GetConfiguration did not succeed.
At line:1 char:1
+ Get-DscConfiguration
+ ~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (MSFT_DSCLocalConfigurationManager:root/Microsoft/...gurationManager) [Get-DscConfiguration], CimException
    + FullyQualifiedErrorId : MI RESULT 1,Get-DscConfiguration

**Comment by SNikalaichyk** [comment]
This issue should be fixed in #98.
But please be aware that the type of the **autoStart** property was changed to **Boolean**.

```
xWebAppPool InstallWebAppPool {
    Ensure = 'Present'
    Name = 'MyAppPool"
    State = 'Started'
    autoStart = $true
    managedRuntimeVersion = 'v4.0'
    managedPipelineMode = 'Integrated'
}
```

**Comment by tysonjhayes** [comment]
This has been merged, I'm going to close this out, please reopen if you are still seeing an issue.
